packages/core/assemblyManager/assemblyManager.ts

```diff
@@ -125,7 +125,7 @@ export default function assemblyManagerFactory(assemblyConfigType: IAnyType) {
           self,
           reaction(
             // have to slice it to be properly reacted to
-            () => getParent(self).jbrowse.assemblies.slice(),
+            () => getParent(self).session.assemblies.slice(),
```

Comment:
this one is interesting, it seems to make an assumption how assemblyManager sits in a tree, and that we always have

root->assemblyManager
root->session

This seems better than looking in

root->jbrowse

But I wonder if there is a way to have a getSession() or something that works here...not gonna worry too much tho

Reply:
I'm glad you pointed this out. I thought this might be better, but I've now realized that `session` could be undefined. I'll change it back.